xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/addnewgroup.vm

```diff
@@ -1,10 +1,16 @@
 <div id="addnewgroup">
-<span style="padding-left:13px;">$services.localization.render('rightsmanager.createnewgroup')</span>
-<form method="post" onsubmit="return false;">
-<div>
-<span style="padding-left:13px;"><input type="text" name="newgroup" value="" id="newgroupi" /></span>
-<span class="buttonwrapper"><input class="button" type="submit" value="$services.localization.render('rightsmanager.creategroup')" onclick="new Ajax.Request(new XWiki.Document(this.form.newgroup.value, 'XWiki').getURL('save', 'form_token=$!services.csrf.token&template=XWiki.XWikiGroupTemplate'), { onFailure: function(transport) { alert('$services.localization.render('rightsmanager.groupexist').replaceAll('"', '&quot;')'.replace('__name__', $('newgroupi').value)); }, onSuccess: function(transport) { window.lb.lbSaveForm(); } });" /></span>
-<span class="buttonwrapper"><input class="button" type="submit" name="formactioncancel" value="$services.localization.render('cancel')" onclick=" window.lb.lbHide(); window.lb.lbClearData(); return false;"/></span>
-</div>
+<form method="post" onsubmit="return false;" class="xform">
+<dl>
+<dt>
+<label for="newgroupi">$services.localization.render('rightsmanager.createnewgroup')</label>
+</dt>
+<dd>
+<input type="text" name="newgroup" value="" id="newgroupi">
+</dd>
+</dl>
+<p>
```

Comment:
"div.buttons" could have been used. 

To be honest, I'm not sure if we should use the: 
- vertical form standard;
- bootstrap standard; 
- or both (.button + .btn classes),
since we still haven't agreed on the standard. 

From a consistency point of view and until we reach a conclusion I prefer the approach you took (the vertical form standard), especially since we extend on the old classes.